.github/workflows/create-tag.yml

```diff
@@ -9,6 +9,10 @@ on:
       tag:
         description: "Tag for new version (1.23.4)"
         required: true
+      skip_deps_check:
+        type: boolean
+        description: "Skip dependencies version check"
```

Comment:
I would add a sentence explaining when it is okay to disable this and make the default `true`.

Reply:
We wanna make this `true`, ie., _**skip**_ the check by default?

Reply:
No, my mistake. Just a comment would be nice though.